apps/api/src/persistence/repositories/data-importer.ts

```diff
@@ -59,7 +59,11 @@ export class DataImporter {
 
         await Promise.all(
             Object.values(ArangoCollectionId)
-                .concat('games' as ArangoCollectionId)
+                .concat(
+                    'games' as ArangoCollectionId,
+                    'term__VIEWS' as ArangoCollectionId,
```

Comment:
Add a TODO to find a solution to this in the near future. We don't want to keep updating this. Should we dynamically register view collections?

Reply:
done